src/sfizz/Voice.cpp

```diff
@@ -63,19 +63,19 @@ void sfz::Voice::startVoice(Region* region, int delay, int number, uint8_t value
     basePan = normalizeNegativePercents(region->pan);
     auto pan { basePan };
     if (region->panCC)
-        pan += normalizeCC(midiState.getCCValue(region->panCC->first)) * normalizeNegativePercents(region->panCC->second);
+        pan += normalizeCC(midiState.getCCValue(region->panCC->first)) * region->panCC->second / 100.0f;
```

Comment:
division is slower, better to multiply in cases of a constant.
In gcc, `fast-math` will resolve this to multiplication, but maybe not so on MS where I believe the fast-math equivalent is not available.